what4/src/What4/Expr/Builder.hs

```diff
@@ -3832,15 +3832,17 @@ foldIndicesInRangeBounds sym f0 a0 bnds0 = do
     _ -> error "internal: invalid index to foldIndicesInRangeBounds"
 #endif
 
-{-
--- | Compute the weighted sum of two bitvectors.
-bvSum :: (1 <= w)
-      => sym
-      -> NatRepr w
-      -> WeightedSum Integer (SymBV sym w)
-      -> IO (SymBV sym w)
-bvSum = undefined
--}
+
+-- | A slightly more aggressive syntaxtic equality check than testEquality,
```

Comment:
... aggressive synta **c** tic equality...  collection **of known** syntax formers.